README.md

```diff
@@ -4,9 +4,7 @@
 https://github.com/user-attachments/assets/cc70c670-2491-4486-871e-f87219344e5d
 
 
-Databased's Missing Semester sessions, inspired from MIT OCW [Missing Semester](https://missing.csail.mit.edu) course, where we learn about basics every developer should know like using command-line, text editors, use fancy features of version control systems, and much more!
-
-We like to call it each session with a fancy session code(based on topics) - DBD001, DBD002, etc.
+Databased's _Missing Semester_ is a series of sessions inspired by MIT's original [Missing Semester](https://missing.csail.mit.edu) but with our own flavour, to introduce undergraduate students with basic tools that they must know as developers but which are never covered in traditional coursework.
 
 ## Sessions
```

Comment:
I feel that names like `MS001`, `MS002`, etc. are better since in the future Databased may do other series too.

Reply:
Can also consider adding names of speakers to each of them (since they are all different).

Reply:
Alright.
But in future someone may take it. So will they overwrite it or something?